src/main/components/urlBar.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+
+const UrlBar = (props) => {
+  const method = (props.AorB === 'B' ? 'POST' : 'GET')
+  const [selected, setSelected] = useState(method);
+
+  function sendFetch () {
+    const sendingObj = { method: selected, mode: 'cors' };
+    if (props.SourceOrDest === 'dest') sendingObj.body = JSON.stringify(props.requestBody);
+
+    // console.log('hello;', document.querySelector('#urlInput').value);
+
+    fetch(document.querySelector('#urlInput').value, sendingObj)
```

Comment:
Ideally, the URI would be stored in state, and update on change so we don’t have to get its value at the time of fetch. (See [React’s form docs](https://reactjs.org/docs/forms.html) for more info)